Document/0x04f-Testing-Network-Communication.md

```diff
@@ -1,51 +1,64 @@
 ## Testing Network Communication
 
-Practically every network-connected mobile app uses the Hypertext Transfer Protocol (HTTP) or HTTP over Transport Layer Security (TLS), HTTPS, to send and receive data to and from remote endpoints. Consequently, network-based attacks (such as packet sniffing and man-in-the-middle-attacks) are a problem. In this chapter we discuss potential vulnerabilities, testing techniques, and best practices concerning the network communication between mobile apps and their endpoints.
+Practically every network-connected mobile app uses the Hypertext Transfer Protocol (HTTP) or HTTP over Transport Layer Security (TLS), HTTPS, to send and receive data to and from 
+remote endpoints. Consequently, network-based attacks (such as packet sniffing and man-in-the-middle-attacks) are a problem. In this chapter we discuss potential vulnerabilities, 
+testing techniques, and best practices concerning the network communication between mobile apps and their endpoints.
```

Comment:
can you please remove these newlines?

Reply:
Oh i see, i'm sorry when i copied the data from my editor to the shell things got missed up and some return to lines got added. i'll fix it now